Tests/MapKitTests/MKMapViewTests.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  MKMapViewTests.swift
+//  SwifterSwift
+//
+//  Created by Hannes Staffler on 24.01.19.
+//  Copyright © 2019 SwifterSwift
+//
+
+import XCTest
+@testable import SwifterSwift
+
+#if canImport(MapKit)
+import MapKit
+import struct CoreLocation.CLLocationCoordinate2D
+
+#if !os(watchOS)
+@available(tvOS 9.2, *)
+final class MKMapViewTests: XCTestCase {
+
+    func testRegister() {
+        let mapView = MKMapView()
+
+        mapView.register(annotationViewWithClass: MKPinAnnotationView.self)
+    }
+
+    @available(iOS 11.0, *, tvOS 11.0, *, macOS 10.13)
+    func testRegisterAndDequeue() {
+        let mapView = MKMapView()
+        let annotation = MKPlacemark(coordinate: CLLocationCoordinate2D(latitude: 0, longitude: 0))
```

Comment:
Why can't you just replace the call with:
```suggestion
        let annotation = MKPlacemark()
```
which does the same thing, and then you can remove the `import`?

Reply:
tried it, but i get a `EXC_BAD_ACCESS` in `dequeueReusableAnnotationView`. Seems like it requires the annotation to have a lat/lng.

Reply:
You're right. I was sure that I ran that in a playground and it worked 🤷‍♂️